supertokens_python/recipe/thirdparty/providers/facebook.py

```diff
@@ -73,6 +73,11 @@ def Facebook(input: ProviderInput) -> Provider:  # pylint: disable=redefined-bui
     if input.config.user_info_map is None:
         input.config.user_info_map = UserInfoMap(UserFields(), UserFields())
 
+    if input.config.user_info_map.from_user_info_api is None:
+        input.config.user_info_map.from_user_info_api = UserFields()
+    if input.config.user_info_map.from_id_token_payload is None:
+        input.config.user_info_map.from_id_token_payload = UserFields()
```

Comment:
why is this required here? given that we aren't adding anything to this in this provider?

Reply:
This comment applies to other providers as well.

Reply:
Fixed.